game_loader.py

```diff
@@ -0,0 +1,91 @@
+from email.policy import default
+from importlib.resources import path
+from operator import le
+from operator import mul
+import os
+import textwrap
+import click
+from pyfiglet import Figlet
+from crpg import generate
+from functools import reduce
+from os.path import exists
+
+#Allows adding specifications when starting program; at the moment, only add the abitly to specify exact file path for .dl
+@click.command()
+@click.option("--path", default="game_libary/", help="Provide file path to a .dl file not located under /game_libary")
+def start(path):
+    print(path)
+    #Possible could be more elegant, checks if user specified path; if path exist runs it
+    if(path == "game_libary/"):
+        dl = selectDL(path)
+    else:
+        defualtChosen=False
+        while((not exists(path)) and (not defualtChosen)):
+            print("Could not locate file, either try again(0) or run default(1)")
+            userIn = getUserInput(2)
+            if(userIn == "1"):
+                defualtChosen=True
+            else:
+                print("Enter file Path")
+                path = str(input())    
+        if(defualtChosen):
+            path = "game_libary/"
+            dl = selectDL(path)
+        else:
+            dl = path
```

Comment:
We could look into how to make this more elegant, like your comment said. 

If it helps, you can add a parameter to make the --path option to be required, that way you can ensure that path always passes something. Then you could use os to check that the directory exists, and then you get a valid directory every time!